### DISCUSSION TITLE
Nethunter

### DISCUSSION THREAD

**Discussion by Kevinbaii** [community/community#41672] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

https://user-images.githubusercontent.com/120519087/207459187-af55d971-9916-4e58-80fd-08a2151f3e81.mp4

Some times it conects witch vnc nethunter but disconects after 5s and always the same message ...vncserver: No matching VNC server running for this user!

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Kevinbaii! The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. I believe reaching out to Nethunter's support or support community would be your best bet, as far as getting an answer to your question. 

Good luck!
